desktop/libs/aws/src/aws/s3/upload.py

```diff
@@ -43,10 +44,79 @@
 else:
   from django.utils.translation import ugettext as _
 
-DEFAULT_WRITE_SIZE = 1024 * 1024 * 50  # TODO: set in configuration (currently 50 MiB)
+DEFAULT_WRITE_SIZE = 1024 * 1024 * 128  # TODO: set in configuration (currently 50 MiB)
 
 LOG = logging.getLogger()
 
+from desktop.lib.exceptions_renderable import PopupException
+from filebrowser.utils import generate_chunks
+
+class S3FineUploaderChunkedUpload(object):
```

Comment:
Same comments for Popupexception and Log messaged and better variable naming just like for new OFS handler above.